src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1837,6 +1837,13 @@ protected function prepareRequestUri()
         } elseif ($this->server->has('REQUEST_URI')) {
             $requestUri = $this->server->get('REQUEST_URI');
 
+            // The REQUEST_URI starts with double slashes.
+            if (0 === strpos($requestUri, '//')) {
+                // We assume that the scheme and host are not provided (e.g. //path/too).
+                // Then patch the call to `parse_url()`.
+                $requestUri = "http://example.com$requestUri";
```

Comment:
@alquerci Looks super weird for me. My idea was to preg_replace original query with something like this "^/*", what do you think about it? It's also weird but this one is weirder for me :)

Reply:
@alquerci This one looks okay for me. Thanks!

Reply:
cc @thomasbisignani

Reply:
I also reproduced this bug, nice catch @arku31! Thanks you for the report.

@arku31 : It looks good to me, thanks you for the fix :+1: